'NODE_ENV' is not recognized as an internal or external command (Note: Resolved with cross-env)

**Issue**:  The server fails to start when running  `npm start` within `./server` 

**Environment**: Windows 10, Git Bash

**Full message**:  `$ npm start`

```bash
'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! create-react-app-server@1.0.0 start: `NODE_ENV=development webpack --watch`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the create-react-app-server@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging
output above.
```

**Resolution**:  I installed the `cross-env` package to the `./server` folder, then updated the scripts to reflect it:

```json
"scripts": {
    "start": "cross-env NODE_ENV=development webpack --watch",
    "start:prod": "cross-env NNODE_ENV=production node build/bundle.js",
    "build":
      "cross-env NNODE_ENV=production webpack -p --progress --profile --bail",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
```

**Recommendation**: Add `cross-env` to the CLI, or utilize an equally valid alternative to running these npm scripts on multiple environments.

Thank you for your work on this project!

--- a/templates/server/package.json
+++ b/templates/server/package.json
@@ -5,22 +5,23 @@
   "main": "src/index.js",
   "license": "MIT",
   "scripts": {
-    "start": "NODE_ENV=development webpack --watch",
-    "start:prod": "NODE_ENV=production node build/bundle.js",
-    "build": "NODE_ENV=production webpack -p --progress --profile --bail",
+    "start": "cross-env NODE_ENV=development webpack --watch",
+    "start:prod": "cross-env NODE_ENV=production node build/bundle.js",
+    "build": "cross-env NODE_ENV=production webpack -p --progress --profile --bail",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "devDependencies": {
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
     "babel-preset-env": "^1.6.1",
+    "cross-env": "^5.1.1",
     "nodemon-webpack-plugin": "0.1.5",
     "null-loader": "^0.1.1",
     "webpack": "^3.8.1",
     "webpack-node-externals": "^1.6.0"
   },
   "dependencies": {
-    "babel-plugin-dynamic-import-node": "^1.1.0",
+    "babel-plugin-dynamic-import-node": "^1.2.0",
     "create-react-app-express": "1.1.0",
     "dotenv": "^4.0.0"
   }

